src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -749,21 +1117,38 @@ describe('LibraryServiceClient', function() {
       var expectedResponse = {};
 
       // Mock Grpc layer
-      client._addLabel = function(actualRequest, options, callback) {
-        assert.deepStrictEqual(actualRequest, request);
-        callback(null, expectedResponse);
-      };
+      client._addLabel = mockSimpleGrpcMethod(request, expectedResponse);
 
       client.addLabel(request, function(err, response) {
         assert.ifError(err);
         assert.deepStrictEqual(response, expectedResponse);
         done();
       });
     });
+
+    it('invokes addLabel with error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var formattedResource = client.bookPath("[SHELF_ID]", "[BOOK_ID]");
+      var label = 'label102727412';
+      var request = {
+          resource : formattedResource,
+          label : label
+      };
+
+      // Mock Grpc layer
+      client._addLabel = mockSimpleGrpcMethod(request, null, error);
+
+      client.addLabel(request, function(err, response) {
+        assert(err instanceof Error);
+        assert.equal(err.code, FAKE_STATUS_CODE);
+        done();
+      });
+    });
   });
 
-  describe('getBigBookAsync', function() {
-    it('invokes getBigBookAsync without error', function(done) {
+  describe('getBigBook', function() {
+    it('invokes getBigBook without error', function(done) {
       var client = libraryV1.libraryServiceClient();
       // Mock request
       var formattedName = client.bookPath("[SHELF_ID]", "[BOOK_ID]");
```

Comment:
The mock response generation logic is shared across languages. I have looked at [that generation code](https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java#L180), it looks like NodeJS may require some extra work for LRO. On the other hand, we may also need to update the sample gen code as well. I have an on-going LRO PR that I will send you for review as soon as it is ready.